app/utils/server/functions/normalizeMessageAttachments.js

```diff
@@ -0,0 +1,16 @@
+import { FileUpload } from '../../../file-upload/server';
+
+export const normalizeMessageAttachments = (message) => {
+	if (message.file && message.attachments && Array.isArray(message.attachments) && message.attachments.length) {
+		const jwt = FileUpload.addJWTToFileUrl({ rid: message.rid, userId: message.u._id, fileId: message.file._id });
```

Comment:
Since the `FileUpload.addJWTToFileUrl` may return `null`, you need to check the returned value before moving forward, don't you think?

Thanks.